samples/tab-ui-templates/ts/.gitignore

```diff
@@ -84,3 +84,23 @@ yarn-error.log*
 npm-debug.log*
 yarn-debug.log*
```

Comment:
Better to commit the original env folder, refer to https://github.com/formulahendry/teamsfx-migrator/tree/main/templates/tab/env

Reply:
added